lexicons/tools/ozone/moderation/getAccountTimeline.json

```diff
@@ -0,0 +1,73 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.moderation.getAccountTimeline",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get timeline of all available events of an account. This includes moderation events, account history and did history.",
+      "parameters": {
+        "type": "params",
+        "required": ["did"],
+        "properties": {
+          "did": {
+            "type": "string",
+            "format": "did"
+          }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["timeline"],
+          "properties": {
+            "timeline": {
+              "type": "array",
+              "items": {
+                "type": "ref",
+                "ref": "#accountTimeline"
+              }
+            }
+          }
+        }
+      },
+      "errors": [
+        {
+          "name": "RepoNotFound"
+        }
+      ]
+    },
+    "accountTimeline": {
+      "type": "object",
+      "required": ["day", "summary"],
+      "properties": {
+        "day": {
+          "type": "string"
+        },
+        "summary": {
+          "type": "array",
+          "items": {
+            "type": "ref",
+            "ref": "#accountTimelineSummary"
+          }
+        }
+      }
+    },
+    "accountTimelineSummary": {
+      "type": "object",
+      "required": ["eventSubjectType", "eventType", "count"],
+      "properties": {
+        "eventSubjectType": {
+          "type": "string",
+          "knownValues": ["account", "record", "chat"]
+        },
+        "eventType": {
```

Comment:
keeping this super generic since we want to be able to blend PLC audit log, mod events and account history from entryway at the moment but also leave room for additional event types in the future.
instead of keeping this flat, we could also make this an object where primary types could be `modEvent`, `plcLog`, `accountHistory` and then have multiple subtypes for each type of event types?

Reply:
Yeah, I think it's a bit difficult a. not having documentation here about which timeline event types may come down, and b. mixing different namespaces such as `'plc_operation'` vs. `'tools.ozone.moderation.defs#modEventTakedown'`.

I'd be pretty satisfied to keep it flat, which is nice and simple, and just continue to use tokens for all the types.  E.g. `'tools.ozone.moderation.defs#timelineEventPlcOperation'` and `'tools.ozone.moderation.defs#modEventTakedown'`.

Reply:
If we're going to use a value like `tools.ozone.moderation.defs#timelineEventPlcOperation` here then it's probably best to document it as a token.